dev_algorithms/wavelengths/template_files/SOAR_Goodman/400_SYZY/README

```diff
@@ -0,0 +1,2 @@
+MasterWaveCalib_A_1_01_M1.fits -- From the Dev suite data (old redux)
```

Comment:
would have thought this was merged in already.  

Maybe I need to submit a PR..

Reply:
Yes I was similarly confused by this.